src/utils/EmailToken.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/no-duplicates */
+import JWT from 'jsonwebtoken';
+import jwt from 'jsonwebtoken';
+import moment from 'moment';
+import dotenv from 'dotenv';
+import models from '../database/models';
+import responseUtil from './responseUtil';
+import strings from './stringsUtil';
+
+dotenv.config();
+
+const ResetToken = user => {
```

Comment:
I believe there is already a function for generating a token. Why don`t you use the same function?